pytest.ini

```diff
@@ -2,3 +2,10 @@
 markers =
```

Comment:
Since you're using markers, I suggest adding `--strict-markers` so it'd be forbidden to use random marker names.

Reply:
I actually don't know what that is there for or if we need it.

Reply:
I'm not familiar with this either, but let's stay focused on the changes Sam is making.

Reply:
Pytest lets you create markers on the fly. So you can use `@pytest.mark.something-undocumented`. It is recommended that only custom markers listed here are allowed, for maintainability.
But I agree that this isn't necessarily related to this PR.